lib/exception_handling/exception_info.rb

```diff
@@ -82,9 +82,10 @@ def honeybadger_context_data
     end
 
     def controller_name
-      @controller_name ||= if @controller
-                             @controller.request.parameters.with_indifferent_access[:controller]
-                           end.to_s
+      @controller_name ||= (
+        @merged_log_context[:honeybadger_grouping] ||
+          (@controller && @controller.request.parameters.with_indifferent_access[:controller])
```

Comment:
You could change this line to just
```
@controller&.request.parameters&.with_indifferent_access[:controller]
```
although I could see leaving it as-is because of the need for `&.`. Just a thought.